<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="suresh-pc" name="Test" tests="3" failures="0" timestamp="29 Jun 2016 09:06:07 GMT" time="28.207" errors="0">
  <testcase name="verifyLoginPageText" time="6.635" classname="Topic20_TestNg_FailedTestcases_Execution.Example1_TestNg"/>
  <testcase name="VerifyForgotPassword" time="4.767" classname="Topic20_TestNg_FailedTestcases_Execution.Example1_TestNg">
    <skipped/>
  </testcase> <!-- VerifyForgotPassword -->
  <testcase name="VerifyForgotPassword" time="2.441" classname="Topic20_TestNg_FailedTestcases_Execution.Example1_TestNg">
    <failure type="java.lang.AssertionError" message="expected [Enter the email address or phone number associated with your account.] but found [Enter the email address or phone number associated with you]">
      <![CDATA[java.lang.AssertionError: expected [Enter the email address or phone number associated with your account.] but found [Enter the email address or phone number associated with you]
	at Topic20_TestNg_FailedTestcases_Execution.Example1_TestNg.VerifyForgotPassword(Example1_TestNg.java:42)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- VerifyForgotPassword -->
</testsuite> <!-- Test -->
